packages/experimental/Drawer/src/Drawer.tsx

```diff
@@ -0,0 +1,41 @@
+/** @jsx withSlots */
+import * as React from 'react';
+import { drawerName, DrawerTokens, DrawerProps, NativeDrawerProps, DrawerType} from './Drawer.types';
+import { compose, UseSlots, buildProps, mergeProps, withSlots } from '@fluentui-react-native/framework';
+import { findNodeHandle} from 'react-native';
+import { ensureNativeComponent } from '@fluentui-react-native/component-cache';
+
+const FRNDrawer = ensureNativeComponent('FRNDrawer');
+
+export const Drawer = compose<DrawerType>({
+  displayName: drawerName,
+  tokens: [drawerName],
+  slots: { root: FRNDrawer },
+  slotProps: {
+    root: buildProps<NativeDrawerProps, DrawerTokens>(() => ({
+      style: {
+      },
+    })),
+  },
+  render: (props: DrawerProps, useSlots: UseSlots<DrawerType>) => {
+    const { target, ...rest } = props;
+    const [nativeTarget, setNativeTarget] = React.useState<number | string | null>(null);
+
+    React.useLayoutEffect(() => {
+      if (typeof target === 'string') {
+        // Pass string type `target` directly
+        setNativeTarget(target);
```

Comment:
Question for my own sake, I know this code was copied from Callout, but what is the actual scenario where a target is a string and not a  ref? I wasn't clear on that..

Reply:
Could be a holdover of older ref behavior? You could use strings as ref in the past, though I believe that's been deprecated: https://reactjs.org/docs/refs-and-the-dom.html#legacy-api-string-refs

(no idea if that's what it's for though, just a guess)

Reply:
My guess too. @PPatBoyd is the one who added the behavior, so I guess I can ask him too.

Reply:
I can address this in my next pr where I actually add the Drawer behavior. I'll remove this code bit for now